eventRouting/event_filter.go

```diff
@@ -0,0 +1,33 @@
+package eventRouting
+
+import (
+	"strings"
+
+	fevents "github.com/cloudfoundry-community/firehose-to-syslog/events"
+)
+
+//EventFilter Given an Event Filter out unwanted event
+type EventFilter func(*fevents.Event) bool
+
+//HasIgnoreField Filter out the event has ignored app filed
+func HasIgnoreField(event *fevents.Event) bool {
+	ignored, hasIgnoredField := event.Fields["cf_ignored_app"]
+	return ignored == true && hasIgnoredField
+}
+
+//NotInCertainOrgs Filter out events not in certain orgs
+func NotInCertainOrgs(orgFilters string) EventFilter {
+	return func(event *fevents.Event) bool {
+		orgName := event.Fields["cf_org_name"]
```

Comment:
What's happen if the field "cf_org_name" do not exist in the event ? Like for Counter, Error type of message?

Reply:
It will not take any change on system metric. This is for LogMessage @shinji62